packages/network/src/network.js

```diff
@@ -108,12 +108,12 @@ const prepareHalfConnection = (zone, { when, watch }, makeAckWatcher) => {
           throw closed;
         }
 
-        return watch(
```

Comment:
I can't see any impact to this change. Nor any of the other changes to network.js. Am I missing something?